.github/workflows/vulnerability-scans.yaml

```diff
@@ -44,6 +44,15 @@ jobs:
           echo "${TAGS_JSON}"
           echo "TAGS=${TAGS_JSON}" >> $GITHUB_OUTPUT
 
+  govulncheck:
+    name: Run govulncheck
+    runs-on: ubuntu-latest
+    steps:
+      - name: Scan for vulnerabilities in go code
+        uses: golang/govulncheck-action@v1
```

Comment:
can you please use the git hash instead of the tag? see the other actions for example

thanks!

Reply:
@cpanato Thank you! I've updated the code